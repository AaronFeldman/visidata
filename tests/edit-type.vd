sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample			columns-sheet		C	
sample_columns	type	キUnits	edit-cell	int	e	
sample_columns		キUnit_Cost	select-row		s	
sample_columns		キTotal	select-row		s	
sample_columns	type		edit-cells	float	ge	
sample_columns			quit-sheet		q	
sample			describe-sheet		I	
