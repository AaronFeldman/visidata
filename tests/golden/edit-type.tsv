column	errors	nulls	distinct	mode	min	max	median	mean	stdev
OrderDate	0	0	43						
Region	0	0	3	Central					
Rep	0	0	11	Jones					
Item	0	0	5	Binder					
Units	0	0	37		2	96	53	49.33	30.08
Unit_Cost	0	0	12	4.99	1.29	275.0	4.99	20.31	47.35
Total	0	0	41		9.03	1879.06	299.4	456.46	447.02
